salesagility/SuiteCRM
# Fixed #716 - The quick create labels at top are not responsive to change

## Description



The quick create labels at top are not responsive to change when i change say the module contact to be persons module. quick create will still say contact

## Motivation and Context

Issue reference: #716 

## How To Test This

1. Check that the quick create drop-down shows the correct values when changing a module name.

## Types of changes


### Final checklist
